bin/deoptigate.create-log.js

```diff
@@ -81,7 +81,7 @@ async function createLog(args, head, simpleHead) {
   const logFile = `${tmpdir()}/deoptigate/v8.log`
 
   const execArgv = [
-      '--trace-ic'
+      '--log-ic'
     , `--logfile=${logFile}`
```

Comment:
We need to keep this compatible with older versions .. just changing this might break compat.
Did you try this in older versions?